src/main/java/pl/simpleascoding/tutoringplatform/domain/user/RoleType.java

```diff
@@ -1,5 +1,5 @@
 package pl.simpleascoding.tutoringplatform.domain.user;
 
 public enum RoleType {
-    ADMIN, USER
+    ADMIN, USER, TEACHER
```

Comment:
We should rename the `USER` role to `STUDENT` instead, but since it's not related to this feature, it will/should be handled in a separate pull request

Reply:
You're right, but I thought that it should be done in commits, that refers to student

Reply:
Agree. We will modify it with "student feature".